assets/src/edit-story/utils/usePreventWindowUnload.js

```diff
@@ -50,8 +50,9 @@ function usePreventWindowUnload() {
     },
     [context]
   );
-
   return setPreventUnload;
 }
 
-export default usePreventWindowUnload;
+const isDevelopment = process.env.NODE_ENV === 'development';
```

Comment:
I'm generally ok with this, but I also think that for a short while this would be useful to spot the history bugs which we had a few. Maybe we can wait for a bit before applying this change?

Reply:
There should be reliable tests in place at least to ensure that this works as expected before implementing this.

I personally definitely prefer my local env to be a copy of the production even if it's a bit annoying. Perhaps there should be a separate variable for disabling this in dev environment which would be off by default but could be turned on for the devs who prefer that.

Reply:
I'll also keep it on - I don't mind it.

But we can simply change these lines to:

```js
const shouldDisablePrevent = Boolean(process.env.DISABLE_PREVENT);
export default shouldDisablePrevent ? () => {} : usePreventWindowUnload;
```

And then you can just run `DISABLE_PREVENT=1 npm run dev` locally if you want to prevent it? Would that be sufficient for your usecase, @merapi?

Reply:
Agree, .env/env var is the way to go here.